specification/ws/src/main/scripts/org/kaazing/specification/ws/opening/request.header.sec.websocket.protocol/handshake.response.rpt

```diff
@@ -26,7 +26,8 @@ read header "Upgrade" /(?i:websocket)/
 read header "Connection" /(?i:Upgrade)/
 read header "Sec-WebSocket-Key" /(?<key>[a-zA-Z0-9+\/=]{24})/
 read header "Sec-WebSocket-Version" "13"
-read header "Sec-WebSocket-Protocol" "primary, secondary"
+read header "Sec-WebSocket-Protocol" "primary"
+read header "Sec-WebSocket-Protocol" "secondary"
```

Comment:
Does k3po keep track of which headers are comma separated headers ? Not all headers are comma separated ones, for e.g even though Date contains values with comma, it is not a comma separated one. Without that knowledge it cannot distinguish between

```
read header "header" "value1"
read header "header" "value2"
```
and

```
read header "header" "value1, value2"
```